lib/modules/betteReddit.js

```diff
@@ -219,8 +219,12 @@ module.go = () => {
 function commentsLinksNewTabs(ele = document.body) {
 	const links = ele.querySelectorAll('.thing div.md a');
 	links::forEachChunked(link => {
-		link.target = '_blank';
-		link.rel = 'noopener noreferer';
+		const { baseURI, href } = link;
+
+		if (!(href.includes(baseURI) && (href.includes('#!settings') || href.includes('#res:settings')))) {
```

Comment:
It would be nice to check if the link's pathname is different from the location (like https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues/2968#issuecomment-250956314 describes), for cases like linking to a particular post listings (nsfw, some subreddit/domain) with the filteReddit settngs already open.  However, we could save that for another time, since the vast majority of settings links should load in the current page.